### PR TITLE
Add PMID file to GitHub lazily

### PR DESCRIPTION
**Pull request by neelsomani**
Needs to be reviewed / tested.

### CODE REVIEW

File: brainspell/json_api.py [hunk 1]
```diff
@@ -323,10 +299,12 @@ def process(self, response, args):
         username = get_github_username_from_api_key(args["key"])
 
         # Get PMIDs that are already added.
-        get_metadata = yield self.github_request(
-            GET, "repos/{0}/{1}/contents/metadata.json".format(
+        route = "repos/{0}/{1}/contents/metadata.json".format(
                 username, get_repo_name_from_collection(
-                    args["collection_name"])), args["github_token"])
+                    args["collection_name"]))
+        print(route)
```
Comment (Sharabesh) [inline brainspell/json_api.py]: rm this?

**Comment by Sharabesh** [comment]
Logically looks good to me, I went through and ran it on Anisha's frontend with around 100 or so PMID's and the hang was no longer there on the create Collections page (at the expense of the viewCollections Page).

**Comment by 62442katieb** [comment]
Does this fix the problem of creating collections with 100s of PMIDs? If so, can we merge & I'll move forward with collection curation?

**Comment by Sharabesh** [comment]
Sorry for the late response, but yes, I did try it out on the ViewCollections page and it did seem to work. A little slow, but definitely no crashes. I'm going to merge for now and we can revert if necessary.

**Pull request merged**